workflow-core/src/commonMain/kotlin/com/squareup/workflow1/BaseRenderContext.kt

```diff
@@ -437,6 +439,8 @@ internal fun <T, PropsT, StateT, OutputT>
     key: String = "",
     handler: (T) -> WorkflowAction<PropsT, StateT, OutputT>
   ) {
-  val workerWorkflow = WorkerWorkflow<T>(workerType, key)
+  val workerWorkflow = workflowTracer.trace({ "Create WorkerWorkflow($workerType,$key)" }) {
```

Comment:
since we're trying to trace otherwise very fast operations, I would suggest to stick to static strings and avoid any string concat to avoid accidentally increasing time. Especially since workerType and key aren't strings already.

Reply:
Done